index.html

```diff
@@ -19,8 +19,8 @@
 <div class="rel_boxtext">
 
 <p>Currently, <a href="//llvm.org">LLVM</a> releases are distributed under the
-<a href="https://llvm.org/docs/DeveloperPolicy.html#new-llvm-project-license-framework">
-Apache License, Version 2.0 with exceptions</a>, an
+<a href="https://raw.githubusercontent.com/llvm/llvm-project/main/llvm/LICENSE.TXT">
+Apache-2.0 with LLVM-exception</a>, an
```

Comment:
I'm not a native English speaker, but to me, it reads more naturally if the word "license" would be added, like this:

Apache-2.0 with LLVM-exception</a> license, an

Otherwise LGTM!

Reply:
Agree - fixed that.